test/graphql.mutation.test.js

```diff
@@ -49,7 +49,11 @@ describe('Mutation Tests', () => {
 
   after(() => sandbox.restore());
 
-  beforeEach('reset db', () => utils.resetTestDB());
+  beforeEach('reset db', async () => {
+    await new Promise(res => setTimeout(res, 500));
```

Comment:
Could we replace that by a try/catch/wait/retry? I'm afaid to slow down all tests by 500ms.

Reply:
try/catch do not fulfill the same function. The problem is that there are actions that continue to need the db even after the test case is over. This 500 millisecond wait gives them time to finish before the db is deleted and recreated. It doesn't happen all the time just in a couple of cases (it's only in graphql.mutation.test.js), the total time increases only in two or three seconds.

Reply:
Ok, hadn't noticed it was only it's only in graphql.mutation.test.js.

I was thinking about.

```
try {
  await utils.resetTestDB()
} catch (e) {
  await new Promise(res => setTimeout(res, 500));
  await utils.resetTestDB()
}
```

Reply:
utils.resetTestDB() doesn't fail, what fails are the processes that were left in the background and over which we can't catch the errors.